include/swupd-build-variant.h

```diff
@@ -11,15 +11,19 @@
 #ifdef SWUPD_WITH_BSDTAR
 #define TAR_COMMAND "bsdtar"
 #define TAR_XATTR_ARGS ""
+#define TAR_XATTR_ARGS_STRLIST
 #else
 #define TAR_COMMAND "tar"
 #define TAR_XATTR_ARGS "--xattrs --xattrs-include='*'"
```

Comment:
@barbieri I was thinking to keep them in order to be used in another kind of declaration but you're right, I can remove unused defines.